src/main/java/domain/lotto/LottoNumber.java

```diff
@@ -0,0 +1,52 @@
+package domain.lotto;
+
+import java.util.Objects;
+
+public class LottoNumber {
+    private static final int MIN_LOTTO_VALUE = 1;
+    private static final int MAX_LOTTO_VALUE = 45;
+
+    private final int value;
+
+    public LottoNumber(final int value) {
+        validateNumber(value);
+        this.value = value;
+    }
+
+    public static LottoNumber of(final String value) {
+        try {
+            int intValue = Integer.parseInt(value);
+            return new LottoNumber(intValue);
+        } catch (NumberFormatException numberFormatException) {
+            throw new IllegalArgumentException("1~45 사이의 번호만 허용합니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException(MIN + "~" + MAX + " 사이의 번호만 허용합니다.");
```

상수를 사용한다면 조건이 변경됐을 때 메시지를 수정하지 않아도 되겠네요!

Reply:
넵 수정해보겠습니다!